client/scss/layouts/_page-editor.scss

```diff
@@ -622,6 +630,204 @@ footer .preview {
   }
 }
 
+.preview-panel {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+
+  &__wrapper {
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  &__iframe {
+    transform-origin: 0 0;
+    width: 100%;
+    display: block;
+  }
+
+  // From DevTools: iPhone SE 375x667
+  &--mobile,
+  &--tablet.preview-panel--has-errors,
+  &--desktop.preview-panel--has-errors {
+    .preview-panel__wrapper {
+      width: calc(375px * min(1, calc(446 / 375)));
+    }
+
+    .preview-panel__iframe {
+      width: 375px;
+      height: calc(100% / min(1, calc(446 / 375)));
+      zoom: min(1, calc(446 / 375));
+      transform: scale(min(1, calc(446 / 375)));
```

Comment:
I don’t understand why we’d need `zoom` and the `scale` transform – is scale not enough?